integration-tests/src/main/java/org/apache/polaris/service/it/test/PolarisRestCatalogViewFileIntegrationTest.java

```diff
@@ -18,20 +18,32 @@
  */
 package org.apache.polaris.service.it.test;
 
+import java.nio.file.Path;
 import java.util.List;
 import org.apache.polaris.core.admin.model.FileStorageConfigInfo;
 import org.apache.polaris.core.admin.model.StorageConfigInfo;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.io.TempDir;
 
 /** Runs PolarisRestCatalogViewIntegrationTest on the local filesystem. */
-public class PolarisRestCatalogViewFileIntegrationTest
+public abstract class PolarisRestCatalogViewFileIntegrationTest
```

Comment:
nit: do we need to change the name to include `base` suffix once we change to abstract?

Reply:
Maybe, however the class was always just a base class and I wanted to limit the changes to a minimum, but make it clear that this is not an actual test-class.